contents.json

```diff
@@ -2043,6 +2043,25 @@
         "http://a2.mzstatic.com/us/r30/Purple1/v4/ae/40/19/ae4019c9-463b-3b73-7147-1b47473596d7/screen696x696.jpeg"
       ]
     },
+    {
+      "title": "Essentia",
+      "category-ids": [
+        "cryptocurrency"
+      ],
+      "tags": [
+        "swift"
+      ],
+      "description": "Multi wallet, All in one Blockchain solution",
+      "license": "gplv3",
+      "source": "https://github.com/essentiaone/Essentia-iOS",
+      "itunes": "https://itunes.apple.com/app/essentia/id1438535734",
```

Comment:
@impul Is the app on review? I just can't reach it because of "The item you've requested is not currently available in the U.S. store.".

Reply:
@impul What about this? In which country it is available? Otherwise, remove this if there is no app in the App Store.

Reply:
@vpeschenkov, an app is available in all App Stores, expect US and China. For example here is a link for UA AppStore: https://itunes.apple.com/ua/app/essentia/id1438535734
Or RU: https://itunes.apple.com/ru/app/essentia/id1438535734